src/System.Text.Json/tests/Serialization/PolymorphicTests.cs

```diff
@@ -14,58 +14,58 @@ public static class PolymorphicTests
         [Fact]
         public static void PrimitivesAsRootObject()
         {
-            string json = JsonSerializer.ToString<object>(1);
+            string json = JsonSerializer.Serialize<object>(1);
             Assert.Equal("1", json);
-            json = JsonSerializer.ToString(1, typeof(object));
+            json = JsonSerializer.Serialize(1, typeof(object));
             Assert.Equal("1", json);
 
-            json = JsonSerializer.ToString<object>("foo");
+            json = JsonSerializer.Serialize<object>("foo");
             Assert.Equal(@"""foo""", json);
-            json = JsonSerializer.ToString("foo", typeof(object));
+            json = JsonSerializer.Serialize("foo", typeof(object));
             Assert.Equal(@"""foo""", json);
 
-            json = JsonSerializer.ToString<object>(true);
+            json = JsonSerializer.Serialize<object>(true);
             Assert.Equal(@"true", json);
-            json = JsonSerializer.ToString(true, typeof(object));
+            json = JsonSerializer.Serialize(true, typeof(object));
             Assert.Equal(@"true", json);
 
-            json = JsonSerializer.ToString<object>(null);
+            json = JsonSerializer.Serialize<object>(null);
             Assert.Equal(@"null", json);
-            json = JsonSerializer.ToString(null, typeof(object));
+            json = JsonSerializer.Serialize((object)null, typeof(object));
```

Comment:
This was the only scenario where having the same API name for the various input arguments caused ambiguity:
The first argument in `JsonSerializer.Serialize(null, typeof(object));` could map to either `object` or to `Utf8JsonWriter`

Reply:
While it's important that `null` works (in case it comes up), I think that people would generally find it easier to write `string s = "null"` than `string s = JsonSerializer.Serialize<object>(null);`, so this isn't a major loss :smile:.  (Yeah, the `null` literal and overloading are sometimes not friends)